fuzz/copy.go

```diff
@@ -0,0 +1,61 @@
+package fuzz
```

Comment:
Suggestion: this `copy.go` file contains only helper methods so for now we could just put them at the end/bottom of `fuzz.go` and do away with this file completely.

Reply:
The helper functions are also referenced in the `corpus.go` file. Although function defined in the `corpus.go` file are of little use right now, this will change in the future. I think it's better to keep the functions where they are.